### PR TITLE
task13

### PR DESCRIPTION
**Pull request by SafaaMagdyAwad**

### CODE REVIEW

File: routes/admin.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\CarController;
+
+
+Route::prefix('car')->group(
```
Comment (Nora-Lootfy) [inline routes/admin.php]: i don't think it have worked for U, need a hint?

**Review by Nora-Lootfy — APPROVED** [review]
All looking good, thanks ya @SafaaMagdyAwad  for trying to apply middleware by yourself, it is very important concept in Laravel

**Comment by SafaaMagdyAwad** [comment]
Yes please,  i need hint

**Comment by Nora-Lootfy** [comment]
By default there is a middleware group applied for file web.php this is what is missing when you create a new custom route file, you don't apply this middleware

Search about default middleware applied for web.php file and apply it to other file and that is it

**Pull request closed without merging**